bin/wayland-launch

```diff
@@ -1,12 +1,14 @@
 #!/bin/sh
-
+set -e
 set -x
 
-if ! snapctl is-connected wayland
-then
-  echo "Wayland interface not connected!"
-  exit 1
-fi
+for PLUG in %PLUGS%; do
+  if ! snapctl is-connected ${PLUG}
+  then
+    echo "${PLUG} interface not connected! Please run: /snap/%SNAP%/current/bin/setup.sh"
+    exit 1
+  fi
+done
```

Comment:
This seems excessive, at least some of the interfaces can be optional, no?

Reply:
Hmm, I was thinking the interfaces connected by the `setup.sh` script would be required. (There can obviously be other interfaces not mentioned to `build-with-plugs.sh`.)

Maybe there should be two build scripts? One for "setup" and one for "launch"?

Or at least, better documentation of the presumptions.

Thoughts?

Reply:
Two scripts sounds excessive, I'd be OK with this if we said `%PLUGS%` _are_ indeed required.